bsdploy/bootstrap-files/pkg.conf

```diff
@@ -1 +1,8 @@
 ASSUME_ALWAYS_YES: YES
+REPO_AUTOUPDATE: NO
```

Comment:
the reasoning is that if enabled each and every pkg installation *always* first downloads ``packagesite.txz`` etc. which is insane IMHO and adds significant network traffic and time delay when bootstrapping and configuring a host.

it's not related to the proxy feature at all, actually, like i said, i was just fed up with it because during the proxy development i needed to bootstrap systems many many times...